test/postman/dbp-test-collection.json

```diff
@@ -898,5 +997,84 @@
 				]
 			}
 		}
+	],
+	"variable": [
+		{
+			"id": "96b5baa4-d3d5-4b18-be9e-448c9e055918",
+			"key": "acsUrl",
+			"value": "{{url}}/alfresco",
+			"type": "string",
+			"description": ""
+		},
+		{
+			"id": "aa64ccbe-797c-4dfd-8011-ca27e8d2b413",
+			"key": "shareUrl",
+			"value": "{{url}}/share",
+			"type": "string",
+			"description": ""
+		},
+		{
+			"id": "8ed8c88b-227e-4727-9215-fadaa9c93837",
+			"key": "apsUrl",
+			"value": "{{url}}/activiti-app",
+			"type": "string",
+			"description": ""
+		},
+		{
+			"id": "6dbd49bb-9839-44ca-9e1e-8c06b256d77d",
+			"key": "identityServiceUrl",
+			"value": "{{url}}/auth",
+			"type": "string",
+			"description": ""
+		},
+		{
+			"id": "c66d9815-e0e2-4fb0-8ec3-5650d14c136b",
+			"key": "identityServiceRealm",
+			"value": "springboot",
```

Comment:
Shouldn't this be grabbed from global with {{IdentityServiceRealm}}

Reply:
I'm not sure I quite understand this comment: this _is_ a global variable.

Reply:
Is this the case for the url variable also? is that a collection variable or global?
If its still global we should turn that to collection also.